pkg/gatewayserver/gatewayserver.go

```diff
@@ -612,7 +612,9 @@ func requireDisconnect(connected, current *ttnpb.Gateway) bool {
 			return true
 		}
 	}
-	if len(connected.Antennas) > 0 && len(current.Antennas) > 0 && connected.Antennas[0].Gain != current.Antennas[0].Gain {
+	if len(connected.Antennas) > 0 &&
```

Comment:
If I start with no antennas (`len(connected.Antennas) == 0`), and update to add an antenna with gain (`len(current.Antennas) > 0`), this won't trigger, right ?

I _think_ it should be `len(connected.Antennas) != len(current.Antennas) || !sameAntennaGain(...)`, such that antenna differences cause a disconnection.

Reply:
Indeed, my bad